target/scripts/helpers/utils.sh

```diff
@@ -11,3 +11,22 @@ function _is_comment
 {
   grep -q -E "^\s*$|^\s*#" <<< "${1}"
 }
+
+# This function parses `/root/.bashrc` and "extracts" the environment
+# variables by making them available to the current shell. After the
+# call, you can work with the environment variables previously defined
+# in `/root/.bashrc` as if you defined them yourself. Note that the
+# file `/root/.bashrc` itself is not altered when you alter one of the
+# parsed environment variables.
+function _get_dms_environment_variables
```

Comment:
As per previous review comment, how is this different/better to `source /etc/dms-settings`?

Reply:
See my comment (I will revert this, as it's not needed).